### DISCUSSION TITLE
pyenv can't change python version in a fresh install of Ubuntu 20.04

### DISCUSSION THREAD

**Discussion by CastilhosR** [pyenv/pyenv#2131] [Q&A] [answered]
On a fresh install of Ubuntu Linux 20.04.02, after installing pyenv, the environments do not work as expected.
Ubuntu Linux 20.04.02, Linux s145 5.11.0-38-generic, pyenv version is 2.1.10, Python version is 3.8.10, GCC version is 9.3.0 

        user@s145:~$ curl https://pyenv.run | bash

In .bashrc file I insert this lines:

    export PYENV_ROOT="$HOME/.pyenv"
    export PATH="$PYENV_ROOT/bin:$PATH"
    
    eval "$(pyenv init -)"
    eval "$(pyenv virtualenv-init -)"

        
In a new shell.

        user@s145:~$ echo @PATH
        /home/user/.pyenv/plugins/pyenv-virtualenv/shims:/home/user/.pyenv/bin:/home/user/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin

        user@s145:~$ pyenv --version
        pyenv 2.1.0
        
        user@s145:~$ pyenv versions
        * system (set by /home/user/.pyenv/version)
          
        user@s145:~$ python3 -V
        Python 3.8.10
        user@s145:~$ type -p python3
        /usr/bin/python3
        
        user@s145:~$ pyenv install 3.10.0
          
        user@s145:~$ pyenv versions
        * system (set by /home/user/.pyenv/version)
          3.10.0
        
        user@s145:~$ pyenv virtualenv 3.10.0 testenv
        user@s145:~$ pyenv versions
        * system (set by /home/user/.pyenv/version)
          3.10.0
          3.10.0/envs/testenv
          testenv
        
        user@s145:~$ pyenv activate testenv
        pyenv-virtualenv: prompt changing will be removed from future release. configure `export PYENV_VIRTUALENV_DISABLE_PROMPT=1' to simulate the behavior.
        
        (testenv) user@s145:~$ python3 -V
        Python 3.8.10        
        user@s145:~$ type -p python3
        /usr/bin/python3`

The venv is not being created/activated ! The new version of Python is not being pointed out.
Who can help me ?

**Comment by CastilhosR** [comment, marked as answer]
Hello ! I found this answer https://github.com/pyenv/pyenv-virtualenv/issues/401#issuecomment-903664058.

**Reply by zamonia500** [reply]
> Hello ! I found this answer [pyenv/pyenv-virtualenv#401 (comment)](https://github.com/pyenv/pyenv-virtualenv/issues/401#issuecomment-903664058).

This comment saved me, thanks!

**Reply by CavalcanteLucas** [reply]
It worked here too! Ty
